e2e/browser/src/components/appContainer/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import {
+  login,
+  logout,
+  handleIncomingRedirect,
+  getDefaultSession,
+} from "@inrupt/solid-client-authn-browser";
+import Notifications from "../notifications/";
+
+const REDIRECT_URL = "http://localhost:3000/";
+const APP_NAME = "Notifications browser-based tests app";
+const issuer = "https://login.inrupt.com/";
+
+const NotificationContainer = () => {
+  if (getDefaultSession().info.isLoggedIn) {
+    return <Notifications />;
+  } else {
+    return <div></div>;
+  }
+};
+
+export default function AppContainer() {
+  const [loggedIn, setLoggedIn] = useState<boolean>(false);
+  const [issuer, setIssuer] = useState<string>("https://login.inrupt.com");
+
+  useEffect(() => {
+    (async () => {
+      const info = await handleIncomingRedirect({
+        restorePreviousSession: true,
+      });
+      if (info !== undefined) {
+        setLoggedIn(info.isLoggedIn);
```

Comment:
You may actually want to store the entire info object, and then pass that down to child components

Reply:
f7acd4ff3913da3214e603aa36015fa3a7f1b138